<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BarrageHistory</class>
 <widget class="QWidget" name="BarrageHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="bottomMargin">
    <number>23</number>
   </property>
   <item>
    <widget class="QWidget" name="BarageHistoryDisplay" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="lastPage">
       <property name="text">
        <string>bottom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pageUpBtn">
       <property name="text">
        <string>Page Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pageDnBtn">
       <property name="text">
        <string>page down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="firstPage">
       <property name="text">
        <string>top</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
